tests/test_table_row_classification.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+feature_table_dict = {
+    "area": np.array([10, 10, 30, 30, 10]),
+    "surface_area": np.array([5, 3, 80, 80, 9])
+}
+feature_table_data_frame = pd.DataFrame(feature_table_dict)
+ground_truth = np.array([1, 1, 2, 2, 1])
+
+
+@pytest.mark.parametrize("feature_table", [feature_table_dict, feature_table_data_frame])
+def test_table_row_classification(tmpdir, feature_table):
```

Comment:
@haesleinhuepf , here I used the `tmpdir` to create a temporary directory in which we write the openCL file. This temporary directory automatically gets cleaned up after the test runs. This is especially nice when running tests locally where you don't want your folder to get polluted with old test files. https://docs.pytest.org/en/6.2.x/tmpdir.html